test/integration/servicebus/queues/client.test.js

```diff
@@ -19,8 +19,7 @@ describe('Queues', function() {
   });
 
   it('should connect, send, and receive a message', function(done) {
-    expect(config.serviceBusHost, 'Required env vars not found in ' + Object.keys(process.env)).to.exist;
-
+    expect(config.address).to.exist;
```

Comment:
seems to me that these could all just go into a `before` right before the whole test suite?